spec/requests/api/site_spec.rb

```diff
@@ -21,6 +23,8 @@
 
       produces 'application/json'
       response '200', 'success response' do
+        Site.preloaded_category_custom_fields << "no_oddjob"
```

Comment:
This is setting a class variable so we need to clean up the state after the test.